### PR TITLE
fix README installation command and PyPI links

### PR DESCRIPTION
**Pull request by Ofekmeister**
The current PyPI install command, links, and images point to the wrong package. I suppose this might be due to the new auto-generated documentation platform in use.

### CODE REVIEW

**Comment by Ofekmeister** [comment]
I've just signed the CLA.

**Comment by dhermes** [comment]
Thanks for the PR but the change was intended. We are moving to `google-cloud`. See #1978 for context

**Comment by Ofekmeister** [comment]
@dhermes Ah, I see. Shouldn't we wait to update install command/links since, as of now, they are incorrect? This may confuse the product's users, especially the error-producing `pip install google-cloud`.

**Comment by dhermes** [comment]
Hopefully it will be transient, i.e. hopefully we have new versions ASAP

**Comment by tseaver** [comment]
We do need to publish a new version of the package Real Soon Now(TM).  Likely a `0.19.0` version for the monolithic package, before we start pushing out releases for the API-specific sub-packages.

**Comment by tseaver** [comment]
@Ofekmeister Thanks for the patch~ I am closing the PR without merging., because as soon as we can get a release out, the links as they stand now (without your patch) will be correct.

**Comment by daspecster** [comment]
@tseaver, I'm reopening from https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2298

File: README.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Quick Start
 
 ::
 
-    $ pip install --upgrade google-cloud
+    $ pip install -U gcloud
```
Comment (dhermes) [inline README.rst]: Leave `--upgrade` there, it's more descriptive.

**Comment by dhermes** [comment]
@Ofekmeister This looks mostly fine other than my comment about leaving `--upgrade` alone (instead of replacing it with `-U`). Also, this change is tiny, so can you squash into one commit?

```bash
$ git checkout master  # that is the name of your branch sent in this PR
$ git branch -m master master-old  # Renames the local master branch
$ git fetch upstream  # Make sure up to date
$ git branch master upstream/master  # Adds a tracking branch for the project's master
$ git checkout master
$ git merge --squash master-old
$ git diff origin/master  # Make sure the changes aren't too crazy.
$ git push --force origin master  # This will overwrite the extra commits in this PR
```

**Comment by tseaver** [comment]
@dhermes For a change this tiny, we could just squash merge it in the Github UI:  I don't think we care about exposing it in a changelog.

**Comment by dhermes** [comment]
@tseaver Good call.

**Comment by Ofekmeister** [comment]
I believe I'm failing with git today. Can I just close this and open a new PR with all changes in one commit?

**Comment by dhermes** [comment]
@Ofekmeister no worries, we can just use the GitHub squash merge feature on the PR. If you'd like to hone your `git` skills I'm happy to help you sort it out via Hangouts chat.

**Comment by Ofekmeister** [comment]
Yes, if you guys could please squash on your end that would be great.

@dhermes Thanks for the offer! That would be enlightening :) Based on search results, I believe it is a Windows issue.

**Comment by tseaver** [comment]
I cancelled the Travis build, as this PR only affects the README.

**Pull request merged**